core/common/src/DisplayStyleSettings.ts

```diff
@@ -706,6 +742,35 @@ export class DisplayStyleSettings {
     return this._modelAppearanceOverrides.size > 0;
   }
 
+  /** Set the planar clip mask for a [[Model]]  drawn by this display style.
+ * @param modelId The ID of the [[model]] whose appearance is to be overridden.

```

Comment:
Has to be a persistent *reality* model, right?

Reply:
Yes  - added that to comment.